<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Macminis-Mac-mini.local" name="Default test" tests="9" failures="1" timestamp="17 Jan 2018 11:16:19 GMT" time="734.329" errors="0">
  <testcase name="Validating_feed1" time="38.454" classname="ANDROID_ARMS.ANDROID_ARMS.FeedAdVAlidationonUI"/>
  <testcase name="Validating_feed2" time="25.705" classname="ANDROID_ARMS.ANDROID_ARMS.FeedAdVAlidationonUI"/>
  <testcase name="Validating_feed3" time="76.205" classname="ANDROID_ARMS.ANDROID_ARMS.FeedAdVAlidationonUI">
    <failure type="java.lang.AssertionError" message="feed3 ad not present">
      <![CDATA[java.lang.AssertionError: feed3 ad not present
	at org.testng.Assert.fail(Assert.java:94)
	at twc.Automation.HandleWithAppium.AppiumFunctions.Check_feed3_ad(AppiumFunctions.java:418)
	at ANDROID_ARMS.ANDROID_ARMS.FeedAdVAlidationonUI.Validating_feed3(FeedAdVAlidationonUI.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Validating_feed3 -->
  <testcase name="Validating_feed4" time="12.466" classname="ANDROID_ARMS.ANDROID_ARMS.FeedAdVAlidationonUI"/>
  <testcase name="Validating_feed5" time="44.838" classname="ANDROID_ARMS.ANDROID_ARMS.FeedAdVAlidationonUI"/>
  <testcase name="Extended_hourly_page_Ad" time="170.203" classname="ANDROID_ARMS.ANDROID_ARMS.FeedAdVAlidationonUI"/>
  <testcase name="Extended_Daily_page_Ad" time="20.397" classname="ANDROID_ARMS.ANDROID_ARMS.FeedAdVAlidationonUI"/>
  <testcase name="Extended_Maps_Radar_page_Ad" time="22.428" classname="ANDROID_ARMS.ANDROID_ARMS.FeedAdVAlidationonUI"/>
  <testcase name="Extended_Newss_page_Ad" time="147.611" classname="ANDROID_ARMS.ANDROID_ARMS.FeedAdVAlidationonUI"/>
</testsuite> <!-- Default test -->
